Sources/Public/DotLottie/DotLottieFileHelpers.swift

```diff
@@ -289,4 +289,65 @@ extension DotLottieFile {
     }
   }
 
+  /// Loads an DotLottie from a data synchronously. Returns a `Result<DotLottieFile, Error>`
+  /// Please use the asynchronous methods whenever possible. This operation will block the Thread it is running in.
+  ///
+  /// - Parameters:
+  ///   - data: The data(`Foundation.Data`) object to load DotLottie from
+  ///   - filename: The name of the lottie file without the lottie extension. eg. "StarAnimation"
+  public static func loadedFrom(
```

Comment:
If you want to include this helper, move it to the `SynchronouslyBlockingCurrentThread` namespace above

Reply:
Sure. Done.